backend/accounts.go

```diff
@@ -257,12 +257,31 @@ func (backend *Backend) nextAccountNumber(coinCode coinpkg.Code, keystore keysto
 		return account.Configurations.ContainsRootFingerprint(rootFingerprint)
 	}
 
-	nextAccountNumber := uint16(len(backend.filterAccounts(accountsConfig, filter)))
```

Comment:
To cover the case of bb01, would this work as well?

```go
if !keystore.SupportsMultipleAccounts() && len(backend.filterAccounts(accountsConfig, filter)) > 0 {
  return 0, ErrAccountLimitReached
}
```

Reply:
yes but the code as it is now also covers it, right?